ban/tests/http/test_municipality.py

```diff
@@ -157,17 +157,29 @@ def test_get_municipality_version(get):
 
 
 @authorize
-def test_can_retrieve_municipality_with_old_insee(get):
+def test_old_insee_should_redirect(get):
     municipality = MunicipalityFactory(insee="12345")
     # This should create a redirect.
     municipality.insee = '54321'
     municipality.increment_version()
     municipality.save()
     # Request with old insee.
     resp = get('/municipality/insee:12345')
-    assert resp.status_code == 200
-    assert resp.json['id'] == municipality.id
-    assert resp.json['insee'] == '54321'
+    assert resp.status_code == 302
+    assert resp.headers['Location'] == 'http://localhost/municipality/insee:54321'  # noqa
+
+
+@authorize
+def test_multiple_redirects_should_return_300(get):
+    MunicipalityFactory(insee="23456")
+    MunicipalityFactory(insee="34567")
+    # Let's say this municipality has been split.
+    IdentifierRedirect.add('Municipality', 'insee', '12345', 'insee', '23456')
+    IdentifierRedirect.add('Municipality', 'insee', '12345', 'insee', '34567')
+    resp = get('/municipality/insee:12345')
+    assert resp.status_code == 300
+    assert '/municipality/insee:23456' in resp.headers['Link']
+    assert '/municipality/insee:34567' in resp.headers['Link']
```

Comment:
I don't think so.
Those multiple redirects will be used for scenarios like a municipality or a parcel split in two (or more) others, and we can't determine one as "primary".